srcpkgs/webkit2gtk/template

```diff
@@ -132,21 +134,132 @@ pre_configure() {
 	esac
 }
 
+post_configure() {
+	mkdir -p webkit2gtk-41
+	pushd webkit2gtk-41
+		configure_args=${configure_args//-DUSE_SOUP2=ON/}
+		configure_args=${configure_args//-DENABLE_WEBDRIVER=OFF/}
+		do_configure
+	popd
+	mkdir -p webkit2gtk-50
+	pushd webkit2gtk-50
+		configure_args+=" -DUSE_GTK4=ON -DENABLE_WEBDRIVER=OFF"
+		do_configure
+	popd
+}
+
+pre_build() {
+	export NINJA_STATUS="[1/3][%f/%t] "
+}
+
+post_build() {
+	export NINJA_STATUS="[2/3][%f/%t] "
+	pushd webkit2gtk-41
+		do_build
+	popd
+	export NINJA_STATUS="[3/3][%f/%t] "
+	pushd webkit2gtk-50
+		do_build
+	popd
+}
+
 post_install() {
+	pushd webkit2gtk-41
+		do_install
+	popd
+	pushd webkit2gtk-50
+		do_install
+	popd
+
 	vlicense Source/WebCore/LICENSE-APPLE
 	vlicense Source/WebCore/LICENSE-LGPL-2.1
 	vlicense Source/WebCore/LICENSE-LGPL-2
 }
 
 webkit2gtk-devel_package() {
+	build_style=meta
+	depends="webkit2gtk40-devel ${sourcepkg}>=${version}_${revision}"
+	short_desc+=" - development files (transitional package)"
+}
+
+webkit2gtk40_package() {
+	depends="${_all_depends}"
+	short_desc="GTK+3 port of the WebKit2 browser engine (soup2)"
+	pkg_install() {
+		vmove usr/lib/webkit2gtk-4.0
+		vmove usr/libexec/webkit2gtk-4.0
+		if [ "$build_option_gir" ]; then
+			vmove "usr/lib/girepository-1.0/*-4.0.typelib"
+		fi
+		vmove "usr/lib/*-4.0.so.*"
+	}
+}
+
+webkit2gtk40-devel_package() {
```

Comment:
This should probably be libwebkit2gtk40 because its primary function is shipping a library (and its supporting content).

Reply:
See comment at webkit2gtk50